### PR TITLE
Ocaml: Should fix 'spawn utop ENOENT'; Ocaml/FSharp: fix support for 'Preloaded' code

### PR DESCRIPTION
**Pull request by spencerwi**
I've updated the `ocaml.js` runner file to call the bash script that sets up environment variables properly; mocha tests still pass, and I can now `docker-compose run ocaml-runner -c 'print_endline "Hello World!";;'` and see output (before I saw no output when I tried to do that).

### CODE REVIEW

**Comment by spencerwi** [comment]
I also noticed when writing an F# kata that I'd neglected the "Preloaded" tab, so I've added support for that in the runners for F# and OCaml.

**Comment by jhoffner** [comment]
I'm getting these errors with ocaml when I build locally:

```
  1) ocaml runner .run should handle basic code evaluation:
     SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
      at exports.runInThisContext (vm.js:53:16)
      at require (internal/module.js:16:19)
      at Object.module.exports.run (lib/runner.js:5:12)
      at Context.<anonymous> (test/runners/ocaml_spec.js:10:20)

  2) ocaml runner .run example challenges should have a passing reference example:
     SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
      at exports.runInThisContext (vm.js:53:16)
      at require (internal/module.js:16:19)
      at Object.module.exports.run (lib/runner.js:5:12)
      at Context.<anonymous> (test/runner.js:28:24)

  3) ocaml runner using oUnit for testing should be able to run a basic test:
     SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
      at exports.runInThisContext (vm.js:53:16)
      at require (internal/module.js:16:19)
      at Object.module.exports.run (lib/runner.js:5:12)
      at Context.<anonymous> (test/runners/ocaml_spec.js:19:20)

```

**Comment by spencerwi** [comment]
Good to know. I saw usage of arrow functions in the runner and assumed ES6. The ocaml and fsharp runners have `let stdin` on ocaml.js:6 and fsharp.js:26 respectively. I fixed, committed, and pushed, but I dunno how that interacts with an already-merged PR.

File: lib/runners/fsharp.js [hunk 1]
```diff
@@ -23,7 +23,13 @@ module.exports.run = function(opts, cb)
     shovel.start(opts, cb, {
         solutionOnly: function (run)
         {
-            var file = util.codeWriteSync('fsharp', opts.solution, dir, 'solution.fsx');
+            let fileContents;
```
Comment (spencerwi) [inline lib/runners/fsharp.js]: This should be `var fileContents;`

File: lib/runners/ocaml.js [hunk 2]
```diff
@@ -3,10 +3,16 @@ var shovel = require('../shovel');
 module.exports.run = function run(opts, cb) {
     shovel.start(opts, cb, {
         solutionOnly: function (run) {
+            let stdin;
```
Comment (spencerwi) [inline lib/runners/ocaml.js]: This should be `var stdin;`

Comment (jhoffner) [inline lib/runners/ocaml.js]: Awesome thanks. Looks like there was also a missing comma in fsharp.js line 59

File: lib/runners/fsharp.js [hunk 3]
```diff
@@ -23,7 +23,13 @@ module.exports.run = function(opts, cb)
...
+            if (opt.setup) {
```
Comment (jhoffner) [inline lib/runners/fsharp.js]: `opt` -> `opts`. I've fixed it in my local branch already.

**Comment by spencerwi** [comment]
Wow, I dunno how I missed those. :-1: Sorry!

**Pull request merged**